crates/webidl/src/util.rs

```diff
@@ -614,7 +614,7 @@ pub fn is_structural(
     item_attrs: Option<&ExtendedAttributeList>,
     container_attrs: Option<&ExtendedAttributeList>,
 ) -> bool {
-    // Note that once host bindings is implemented we'll want to switch this
+    // Note that once component model is implemented we'll want to switch this
```

Comment:
```suggestion
    // Note that once host bindings is implemented we'll want to switch this
```
I honestly don't know what the intentions here were, but I wouldn't update it just to make it easier to use `git blame` (until we not only figure out what was meant here, but actually also figure out if this is still correct or not).